packages/react-flex/tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "dist",
+    "target": "es5",
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strictNullChecks": true,
+    "noImplicitAny": true,
```

Should be like this. Would you mind making the change in the templates too? (scripts/create-package/plop-templates-react/tsconfig.json and scripts/create-package/plop-templates-node/tsconfig.json)
```suggestion
    "strict": true,
```

---

sure!